tox.ini

```diff
@@ -0,0 +1,13 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
```

Comment:
I would suggest to merge this file with `pyproject.toml` ([documentation](https://tox.readthedocs.io/en/latest/example/basic.html#pyproject-toml-tox-legacy-ini)) to avoid clutter.